auth-web/src/components/auth/manage-business/StartNewBusinessHelp.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <section class="start-new-business-help">
+    <h3 class="text-center">
+      Help with Starting and Managing a Business
+    </h3>
+    <div class="mt-10">
+      <p class="ml-1">
+        Start a named or numbered business in B.C. by following these steps:
+      </p>
+      <ol class="bold">
+        <li class="mt-8">
+          <h4>Decide on a Business Type</h4>
+          <ul class="mt-4 text-align ml-n6">
+            <li>
+              Decide which business structure is the most appropriate for you. A few options are: a sole
+              proprietorship, partnership, or corporation. Each structure has different legal and financial
+              implications. <br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+              <a href="https://entity-selection-prod.apps.silver.devops.gov.bc.ca/">
+                Use the Business Structures Wizard to help you decide.</a><v-icon
+                color="primary"
+                class="small-icon mb-1"
+                size="16"
+              >
+                mdi-open-in-new
+              </v-icon>
+            </li>
+            <li>
+              If you want to start a corporation, you also have the choice of using a named or numbered company.
+            </li>
+            <li>
+              Once your Name Request has been approved you must return to this screen to Incorporate or Register your
+              business using your approved Name Request.
+            </li>
+          </ul>
+        </li>
+
+        <li class="mt-10">
+          <h4>Request a Business Name or Use a Numbered Company</h4>
+          <ul class="mt-4 ml-2 ml-n6 text-align">
+            <h4 class="mb-5">
+              Request a Business Name
+            </h4>
+            <li>
+              To start a named business in B.C., change the business name of an existing company, or
+              merge two or more companies using a new name, click the
+              <strong>"Get Started with a B.C. Based Business"</strong> button on this
+              screen and follow the instructions.
+            </li>
+            <li>
+              Before registering or incorporating a named business, you will need to first submit a Name Request.
+              You will be asked to create a unique name for your business and submit your name(s) for review by BC
+              Registries staff.
+            </li>
+            <li>
+              Once you submit your Name Request you can return to this screen and your Name Request (NR) will
+              automatically appear in your table.
+            </li>
+            <li>
+              If you do not see your Name Request in your table (e.g. if you submitted your Name Request without
+              being logged into your BC Registries account), you can add your Name Request manually by looking up
+              your business name or your NR &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;number.
```

Comment:
Why did you add all these non-blanking spaces?

Reply:
To adjust the text alignment. Otherwise the link text looked broken and misaligned

Reply:
Wrong solution to the problem. And I can break it (and other sentences) by making my screen narrowing and forcing sentence wrapping.

Reply:
It's probably related to your li / marker / :before CSS. Maybe that could be simplified?

Reply:
Okay taking a look at that

Reply:
The new code looks great, and it works perfectly! Nice! Well done 👍